src/TypeScript/Sarif.SDK/package-lock.json

```diff
@@ -22,13 +22,13 @@
       "integrity": "sha1-+/bAeDaVRr0V/6eLp/Py4XBIB2k=",
       "dev": true,
       "requires": {
-        "@types/node": "8.10.9"
+        "@types/node": "*"
       }
     },
     "@types/lodash": {
-      "version": "4.14.107",
-      "resolved": "https://registry.npmjs.org/@types/lodash/-/lodash-4.14.107.tgz",
-      "integrity": "sha512-afvjfP2rl3yvtv2qrCRN23zIQcDinF+munMJCoHEw2BXF22QJogTlVfNPTACQ6ieDyA6VnyKT4WLuN/wK368ng==",
+      "version": "4.14.120",
```

Comment:
>.14.12 [](start = 19, length = 6)

Security report states the we should be updating to 4.17.11. This release occurred on Sept. 2018 according to github tag.

Reply:
That package contains the type definitions and doesn't have a synced version #. This is the latest version of this package. [https://www.npmjs.com/package/@types/lodash](https://www.npmjs.com/package/@types/lodash)

---
In reply to: [255229325](https://github.com/Microsoft/sarif-sdk/pull/1270#discussion_r255229325) [](ancestors = 255229325)

Reply:
You know this area better than I. I believe you're saying that the releases below are type definition releases only, not the shipped NPM version? And that the versioning of these type definitions is not in sync with the NPM release? I am only pushing on this because the github.com security alert specifically referenced 4.17.11 as required. Marking this matter as approved, if you're confident of how to place the releases below, proceed. We'll see if we get flagged again or not. :)

[https://github.com/lodash/lodash/releases](https://github.com/lodash/lodash/releases) 



---
In reply to: [255230735](https://github.com/Microsoft/sarif-sdk/pull/1270#discussion_r255230735) [](ancestors = 255230735,255229325)

Reply:
The update for the lodash package that got flagged is down on line 449.

---
In reply to: [255305614](https://github.com/Microsoft/sarif-sdk/pull/1270#discussion_r255305614) [](ancestors = 255305614,255230735,255229325)